src/messages/ru/podium/view.php

```diff
@@ -341,6 +341,7 @@
     'Sign in' => 'Войти',
     'Sign in to reply' => 'Залогиньтесь, чтобы ответить',
     'Sign out' => 'Выйти',
+    'Signature' => 'Подпись',
```

Comment:
This text has been removed in previous commit. There is 'IP' there instead (admin > logs). 
Please remove this line.